cmd/restic/cmd_backup.go

```diff
@@ -557,7 +557,10 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 	arch.SelectByName = selectByNameFilter
 	arch.Select = selectFilter
 	arch.WithAtime = opts.WithAtime
-	arch.Error = p.Error
+	arch.Error = func(item string, fi os.FileInfo, err error) error {
+		arch.Success = false
```

Comment:
Please use a local variable. It's no good to have to look at _both_ `cmd_backup.go` and `archiver.go`.